apps/rebar/src/rebar_prv_manifest.erl

```diff
@@ -73,6 +74,8 @@ do(State) ->
 -spec format_error(any()) -> iolist().
 format_error({format_not_supported, Format}) ->
     io_lib:format("Format '~p' is not supported. Try 'erlang' or 'eetf'.", [Format]);
+format_error(no_json_module) ->
+    io_lib:format("The 'json' module is not available. Either upgrade to OTP 27 or use a different format.", []);
```

Comment:
"Either upgrade to OTP 27 or newer, or select a different output format." maybe

Reply:
Fixed the spelling, thanks.